src/cascadia/TerminalControl/InteractivityAutomationPeer.cpp

```diff
@@ -118,7 +118,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         return WrapArrayOfTextRangeProviders(pReturnVal);
     }
 
-    XamlAutomation::ITextRangeProvider InteractivityAutomationPeer::RangeFromChild(XamlAutomation::IRawElementProviderSimple childElement)
+    XamlAutomation::ITextRangeProvider InteractivityAutomationPeer::RangeFromChild(XamlAutomation::IRawElementProviderSimple /*childElement*/)

```

Comment:
Idk why the build is complaining about this _now_. Happy to pull it out into its own PR, if desired.

Reply:
Ah it's because of a new VS version on the builds agents

Reply:
Makes sense! I'll make a separate PR for these changes then and focus on that for a bit.